grpc/server/dronecore_server.cpp.in

```diff
@@ -58,32 +60,35 @@ void RunServer()
     std::vector<::grpc::Service *> list;
 ${PLUGIN_MAP_APPEND_STRING}
 
+    CommonRPCImpl* common_service = new CommonRPCImpl(&dc);
+
     while (file >> plugin) {
         auto service_obj = map[plugin](&dc);
         list.push_back(service_obj);
     }
 
-    bool discovered_device = false;
-    DroneCore::ConnectionResult connection_result = dc.add_udp_connection();
+    uint16_t discovered_device = 0;
```

Comment:
I don't think `uint16_t` is the right type here. The UUID is `uint64_t`. And otherwise, it makes sense to just use `int.